core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/worker/DPThread.scala

```diff
@@ -54,6 +54,9 @@ class DPThread(
 
   var backpressureStatus = false
 
+  // Pass internal queue to input manager for reader threads to use
+  dp.inputManager.inputMessageQueue = internalQueue
```

Comment:
i thought you said you put this into constructor (i.e., the `__init__` method).

Reply:
Yes. I forgot. I have moved this into the constructors.